operator/internal/manifests/internal/sizes.go

```diff
@@ -221,6 +242,11 @@ var StackSizeTable = map[lokiv1beta1.LokiStackSizeType]lokiv1beta1.LokiStackSpec
 			QueryFrontend: &lokiv1beta1.LokiComponentSpec{
 				Replicas: 1,
 			},
+			Ruler: &lokiv1beta1.LokiRulerComponentSpec{
+				LokiComponentSpec: lokiv1beta1.LokiComponentSpec{
+					Replicas: 1,
```

Comment:
Change to zero so that the default deployment will be without a ruler component. And make sure that this can be overwrite by some external configuration, such as with the example file from hack

Reply:
I would say that it would be better to add a flag like the other optional components.

In addition, for the `small` and `medium` sizes, the replica count should probably set to `2`.

Reply:
@Red-GV can you elaborate more about this flag? and as @eranra suggested the default would be 0?